content/technotes/2019-10-09-cran-checks-api-update.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: "cran-checks-api-update"
+title: "cran checks API: an update"
+date: 2019-10-09
+authors:
+  - Scott Chamberlain
+categories:
+  - technotes
+topicid:
+tags:
+- R
+- API
+- cran
+- infrastructure
+---
+
+If you have an R package on CRAN, you probably know about CRAN checks. Each package on CRAN, that is not archived on CRAN, has a checks page, like this one for `ropenaq`:
```

Comment:
noob comment that might help others?

In
> Each package on CRAN, that is not archived on CRAN

I find "that is not archived on CRAN" confusing. I think here "archived" means it's no longer on CRAN, but "archive" is in CRAN's name. Can you do "Each package on CRAN, that is not archived on CRAN (explain meaning here), ..."

Reply:
added a footnote